eth/protocols/eth/broadcast.go

```diff
@@ -37,12 +37,26 @@ type blockPropagation struct {
 	td    *big.Int
 }
 
+// blockPropagation is a block propagation event, waiting for its turn in the
+// broadcast queue.
+type blockAndBlobPropagation struct {
```

Comment:
Is possible to reuse blockPropagation?
such things like
```
type blockPropagation struct {
	block *types.Block
	td    *big.Int
	sidecars types.BlobTxSidecars `rlp:"optional"`
}
```

Reply:
same question, we can also remove `AsyncSendNewBlockAndBlob(...)` if we reuse `blockPropagation`

Reply:
Removed `AsyncSendNewBlockAndBlob()` in this commit: https://github.com/bnb-chain/bsc/pull/2253/commits/10cfa610e6dd906098f087de52d64db78859ea64